scripts/react_build.sh

```diff
@@ -0,0 +1,37 @@
+# Script to build the Foursight React code:
+# - Use npm to do basic React build in the 'react' sub-directory (of this main repo).
+# - Change the generated hashed files names and css files (e.g. main.07ce1b0d.js
+#   and main.07ce1b0d.css) to the unadorned file names (e.g. main.js, main.css).
+# - Copy the resultant files into the 'react' sub-directory of the 'foursight_core'
+#   sub-directory; these are to be checked into GitHub.
```

Comment:
Why? Shouldn't they be built on demand? Or is there no point once it is built?

Reply:
Relates to above comment about whether these "compiled" should should be checked in. If so then if we didn't do this the names of the checked in files would change on every build. Though that fact of this itself kinda suggests to me that maybe we shouldn't be checking these in.